### ISSUE TITLE
JSON / XML file for hardcoded values

### ISSUE THREAD

**Issue by justgo97** [rehlds/ReGameDLL_CS#286]
so the file will have all hardcoded values models , sounds...ect and if the file is missing or uncomplete the default values will be used and it should be loaded on map start

**Comment by peon501** [comment]
json would be better.

**Comment by justgo97** [comment]
Yeah or that too the important thing is to get rid of all hardcoded values

**Comment by peon501** [comment]
To be honest that would be massive change.

**Comment by peon501** [comment]
Iam not sure if this should be mainstream.

**Comment by peon501** [comment]
Also 1 file would be bad idea. Should be different files for sounds, models, sprites

**Comment by peon501** [comment]
also uncomplete file is better for specific models and so on. why should u have large file with all stuff ?

**Comment by peon501** [comment]
also maybe would be better hold as simple as in cfg files.
key value

**Comment by In-line** [comment]
We already have cvars.

**Comment by justgo97** [comment]
> We already have cvars.

Yeah, but i m talking about somthing for models/sounds...ect somthing like these to easy edit hardcoded stuff : 

```
{
    "PlayerModels": [
        // Player models location is "models/player/model_name/model_name.mdl"
        {
            "TERRORIST": {
                "MODEL_TERROR": "terror",
                "MODEL_LEET": "leet",
                "MODEL_ARCTIC": "arctic",
                "MODEL_GUERILLA": "guerilla"
            },
            "CT": {
                "MODEL_URBAN": "urban",
                "MODEL_GSG9": "gsg9",
                "MODEL_SAS": "sas",
                "MODEL_GIGN": "gign"
            }
        }
    ]
}
```

**Comment by Nord1cWarr1or** [comment]
As summarized by the collaborator in the thread: "We already have cvars" — model/sound customization is already achievable through existing cvars and hookchains (e.g. `GetCSModelName` paths / ReAPI), and an external JSON/XML config layer for all hardcoded values was acknowledged by the participants as a massive change with no maintainer support. Closing as not planned.
